README.md

```diff
@@ -1,8 +1,65 @@
-This project requires an up-to-date version of Python 3. It also uses pipenv to manage packages.
+# Testing Ryanair Project
+**PythonTestAutomationFramework**
 
-To set up this project on your local machine:
+**Description**
+
+Test Automation Framework for web automation of "www.ryanair.com" using Selenium and Python using BDD testing approach with the below features:
+
+* Framework is based on Page Object Model. 
+* BDD approach (Pytest-BDD plugin) with Scenarios Outline
+* Reporting using Allure report.
+* Logging to external file.
+
+    
+The project is being developed during the iTechArt internship.
+
+# How to install it
+Make sure you have python3.10 installed on your machine by typing in cmd ``python3 --version`` if not - go to https://realpython.com/installing-python/#step-1-download-the-python-3-installer.
+
+1) Clone the repository to any local path.
+
+``$ git clone https://github.com/EgorSolenok/TestRyanair.git``
+
+2) Make sure you have  allure command line  by typing in cmd ``allure --version``. If not - you have to install allure command line and add the allure folder installation into system environment variable: https://docs.qameta.io/allure/#_installing_a_commandline
+
+3) Make sure you have pipenv  by typing in cmd ``pipenv --version``. If not - you have to install pipenv for creation virtual environment and installation packages: https://pipenv.pypa.io/en/latest/  
+
+``$ pip install --user pipenv``
+
+4) Install dependencies:
+
+``pipenv install``
+
+
+# How to run it
+
+**For all tests runs you should to type commands in project directory.**
+**You should type your actual credentials to commandline instead of < USER@EMAIL.COM > and < PASSWORD > in every case (without brackets).**
+
+
+To run scenario steps in feature "Verify user is finding flights in specified dates" and create report:
+
+`` pipenv run python -m pytest -k  "flights_search" -v --tb=line --user=<USER@EMAIL.COM> --password=<PASSWORD> 
```

Comment:
without test creds it is not possible to run the tests

Reply:
Is there a better way to put sensible data to git? Creds will send with PR next time to reviewer